### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by garfinkels-dnb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,31 +4,72 @@
...
     <script type="text/javascript">
-        var apiKey = '';
+        const url = new URL(window.location.href);
 
-        setInterval(() => {
-            if (apiKey) {
-                console.log('Firing pixel with API Key: ' + apiKey);
+        const coretag = window.document.createElement("script");
+        coretag.type = "text/javascript";
+        coretag.src = `https://cdn-0.d41.co/tags/dnb_coretag_v5${url.searchParams.has('prod') ? '' : '_rc'}.min.js`
```
Comment (slinlattice) [inline index.html]: Prefer not to decide env with query parameter. It's very easy for QA to make mistake. 
Either use a dropdown or keep duplicate copies.

Comment (garfinkels-dnb) [reply index.html]: 07e2912dfb6ebb8455ce2339ef11fe716c0ea418

File: index-prod.html [hunk 2]
```diff
@@ -4,42 +4,54 @@
...
+
+        window.addEventListener('load', () => {
+            render();
+        });
     </script>
 </head>
 <body>
-    <h1>(PROD) Pixel Fire Test</h1>
+<div>
+    <h1>(QA) Pixel Fire Test</h1>
```
Comment (slinlattice) [inline index-prod.html]: change this back to prod.

**Review by slinlattice — APPROVED** [review]
LGTM. typo in prod file heading. 
Thank you!

**Pull request merged**